titus_isolate/config/constants.py

```diff
@@ -44,6 +44,11 @@
 REMOTE_FREE_ALLOCATOR = 'REMOTE_FREE_ALLOCATOR'
 REMOTE_REBALANCE_ALLOCATOR = 'REMOTE_REBALANCE_ALLOCATOR'
 
+# New Remote Allocator
+GRPC_REMOTE_ALLOC_ENDPOINT = 'TITUS_ISOLATE_GRPC_REMOTE_ALLOCATOR_ENDPOINT'
```

Comment:
You probably want all 3 of these in the "restart" list at the bottom of this file.